### PR TITLE
#5425 continued

### PR DESCRIPTION
**Pull request by stuartarchibald**
Continues #5425

Closes #5425

### CODE REVIEW

**Comment by gmarkall** [comment]
I've added this to the 0.52RC milestone for now - unless it's critical for 0.51.1?

**Comment by stuartarchibald** [comment]
> I've added this to the 0.52RC milestone for now - unless it's critical for 0.51.1?

0.52RC is fine, thanks.

File: numba/tests/test_withlifting.py [hunk 1]
```diff
@@ -522,15 +522,21 @@ def test_case07_mystery_key_error(self):
...
+        exstr = str(raises.exception)
         self.assertIn(
-            "missing type annotation on outgoing variables",
-            str(raises.exception),
-            )
+            "Missing type annotation on outgoing variable(s): ['t', 'u']",
+            exstr)
+        self.assertIn("Example code: with objmode"
+                      "(t='<add_type_as_string_here>')",
+            exstr)
```
Comment (sklam) [inline numba/tests/test_withlifting.py:539]: odd indentation here---i didn't see `exstr` at first.  can the string "Example code..." be move to the same level as `exstr`?

Comment (stuartarchibald) [reply numba/tests/test_withlifting.py]: done in 210a8f2

**Pull request merged**